src/lib/requests/thirdpartyRequests.js

```diff
@@ -12,9 +12,9 @@ class ThirdpartyRequests extends BaseRequests {
         return this._post('authorizations', 'authorizations', authorizationBody, destParticipantId);
     }
 
-    async getThirdpartyRequestsTransactions(transactionRequestId, destParticipantId) {
+    async getThirdpartyRequestsTransactions(transactionRequestId) {
         const url = `thirdpartyRequests/transactions/${transactionRequestId}`;
-        return this._get(url, 'thirdparty', destParticipantId);
+        return this._get(url, 'thirdparty');
```

Comment:
@kleyow & @lewisdaly do we really want to drop fspiop-destination header? How will we handle the case when multiple pisp will be onboarded?